scope.go

```diff
@@ -493,7 +494,8 @@ func (scope *Scope) scan(rows *sql.Rows, columns []string, fields []*Field) {
 		}
 
 		for fieldIndex, field := range selectFields {
-			if field.DBName == column {
+			// case insensitive match on field which is required for oracle
+			if strings.EqualFold(field.DBName, column) {
```

Comment:
Will this break other databases since [not all of them are case insensitive](https://stackoverflow.com/questions/153944/is-sql-syntax-case-sensitive)?